src/styles/getModalStyles.js

```diff
@@ -96,6 +96,24 @@ export default (type, windowDimensions) => {
             animationIn = 'fadeInLeft';
             animationOut = 'fadeOutLeft';
             break;
+        case CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED:
+            modalStyle = {
+                margin: 0,
```

Comment:
NAB, pretty much every modal has this `margin: 0` on it. I wonder if this should just be a default for all of them.